src/de.rs

```diff
@@ -285,7 +296,7 @@ impl<'de, 'a, R: Read> de::Deserializer<'de> for &'a mut Deserializer<R> {
     }
 }
 
-struct SeqAccess<'a, R: Read + 'a> {
+pub struct SeqAccess<'a, R: Read + 'a> {
```

Comment:
All `_Access` types are internal to crate, so we can make them `pub(crate)`

Reply:
Nice!